lib/shopify_transporter/pipeline/magento/order/line_items.rb

```diff
@@ -42,11 +42,12 @@ def line_item(item)
           end
 
           def tax_lines(item)
-            {
+            # Magento supports only one type of tax, the tax_lines should always be in array format
+            [{
```

Comment:
This is minor, if our linter did not catch it then don't worry about changing it. I prefer if we split out the `[ &  {` onto separate lines here and in the test:
```
  [
    {
      ...
    }
  ]
```

Reply:
Do we need the comment btw? I think this was ultimately confusion on our end but may be obvious to others. I'd vote for leaving the comment out.

Reply:
👍 to removing the comment - in general we try to avoid comments except for exceptional circumstances. In this case the test documents the expected format as well.